src/common/form.js

```diff
@@ -159,6 +159,7 @@ export class Toggle extends Component {
 
   set value (value) {
     this.refs.input.checked = Boolean(value)
+    this.forceUpdate()
```

Comment:
Without that, the icon did not switch. Assigning a ref should trigger a rendering ?